common/static/js/vendor/draggabilly.pkgd.js

```diff
@@ -1748,7 +1748,9 @@ return Unidragger;
 
   if ( typeof define == 'function' && define.amd ) {
     // AMD
-    define( [
+    define(
```

Comment:
Reviewers: this is the key problem which is that the Draggabilly vendor file doesn't correctly define its own module when using RequireJS. This fix allows the file to be bundled by RequireJS Optimizer.